cpp/util/etcd.cc

```diff
@@ -628,6 +639,8 @@ void EtcdClient::WatchState::RequestDone(Status status,
   {
     // This is probably due to a timeout, just retry.
     if (!status.ok()) {
+      LOG(INFO) << "Watch request failed (" << status << "), will retry after "
```

Comment:
INFO? WARNING? One of those is evil because of flushes, right? I guess this is INFO enough (since we're retrying it)...

It'd be nice to have the log messages consistent for these errors, though. I don't think we need to mention when we'll retry?